client/src/modules/CourseManagement/components/ExpelledStudentsStats.tsx

```diff
@@ -0,0 +1,237 @@
+  otherComments: string;
+  submittedAt: string;
+}
+
+interface UseRequestResultWithRefetch {
+  data: DetailedExpelledStat[] | undefined;
+  error: Error | undefined;
+  loading: boolean;
+  refetch: () => void;
+}
+
+const fetchExpelledStats = async (): Promise<DetailedExpelledStat[]> => {
+  const response = await fetch('/api/course/stats/expelled');
+  if (!response.ok) {
+    throw new Error('Failed to fetch stats');
+  }
+  return response.json() as Promise<DetailedExpelledStat[]>;
+};
+
+const ExpelledStudentsStats: React.FC = () => {
+  const { data, error, loading, refetch } = useRequest(fetchExpelledStats) as unknown as UseRequestResultWithRefetch;
+  const [isDeleting, setIsDeleting] = React.useState(false);
+
+  const handleDelete = async (id: string) => {
+    setIsDeleting(true);
+    try {
+      const response = await fetch(`/api/course/stats/expelled/${id}`, {
+        method: 'DELETE',
+      });
+      if (!response.ok) {
+        throw new Error('Failed to delete stat');
+      }
+      if (typeof refetch === 'function') {
+        refetch();
+      }
+    } catch (err) {
+      console.error('Error deleting stat:', err);
+    } finally {
+      setIsDeleting(false);
+    }
+  };
+
+  const columns: ColumnsType<DetailedExpelledStat> = [
+    {
+      title: 'Course',
+      dataIndex: ['course', 'name'],
+      key: 'courseName',
+      render: (_text, record) => (
+        <div>
+          <PublicSvgIcon size="25px" src={DEFAULT_COURSE_ICONS[record.course.logo]?.active} />
+          <Text strong style={{ marginLeft: 15 }}>
+            {record.course.alias}
+          </Text>
+          <br />
+          <Text type="secondary">{record.course.fullName || record.course.name}</Text>
+        </div>
+      ),
+    },
+    {
+      title: 'Student GitHub',
+      dataIndex: ['user', 'githubId'],
+      key: 'githubId',
+      render: githubId => (
+        <a href={`https://github.com/${githubId}`} target="_blank" rel="noopener noreferrer">
+          {githubId}
+        </a>
+      ),
+    },
+    {
+      title: 'Reasons for Leaving',
+      dataIndex: 'reasonForLeaving',
+      key: 'reasons',
+      render: (reasons: string[]) => {
+        if (!reasons) {
+          return null;
+        }
+        return (
+          <>
+            {reasons.map(reason => (
+              <Tag key={reason}>{reason.replace(/_/g, ' ')}</Tag>
+            ))}
+          </>
+        );
+      },
+    },
+    {
+      title: 'Other Comments',
+      dataIndex: 'otherComments',
+      key: 'otherComments',
+    },
+    {
+      title: 'Date',
+      dataIndex: 'submittedAt',
+      key: 'date',
+      render: dateUtcRenderer,
+    },
+    {
+      title: 'Action',
+      key: 'action',
+      render: (_text, record) => (
+        <Button danger onClick={() => handleDelete(record.id)} loading={isDeleting}>
+          Del
+        </Button>
+      ),
+    },
+  ];
+
+  if (error) {
+    return <p>Failed to load statistics.</p>;
```

Comment:
```suggestion
    return <Typography.Paragraph>Failed to load statistics.</Typography.Paragraph>;
```
or
```suggestion
    return <Typography.Text>Failed to load statistics.</Typography.Text>;
```
depending on your needs